### PR TITLE
Updating Copyright Notices for 2023.

### PR DESCRIPTION
**Pull request by sihyung-maxim**
The ME13B and ME55 files will remain *mostly untouched in this PR. Their copyright notices will be updated in their respective development branches.

### CODE REVIEW

**Comment by sihyung-maxim** [comment]
I'll be updating the copyright notices to the 2023 version for the MSDK. There won't be any significant changes, but there will be a lot of affected files to review. I don't expect you to review thousands of files as I've already updated most of the SDK files to the 2022 version last year, but a spot check would be appreciated.

**Comment by EdwinFairchild** [comment]
Can you merge main into this to get the latest merged PR, to see if it stops jobs running on draft PR

**Review by Jake-Carter — APPROVED** [review]
Looks good.  Also updated the Makefile template in MSDKGen:  https://github.com/Analog-Devices-MSDK/MSDKGen/commit/4a2dd0bc95dedd4a9510660976527aed265318e2

File: Examples/MAX78000/CNN/digit-detection-demo/TFT/fthr/bmp/bmp2c.py [hunk 1]
```diff
@@ -1,11 +1,35 @@
-###################################################################################################
-#
-# Copyright (C) 2020-2021 Maxim Integrated Products, Inc. All Rights Reserved.
-#
-# Maxim Integrated Products, Inc. Default Copyright Notice:
-# https://www.maximintegrated.com/en/aboutus/legal/copyrights.html
-#
-###################################################################################################
```
Comment (sihyung-maxim) [inline Examples/MAX78000/CNN/digit-detection-demo/TFT/fthr/bmp/bmp2c.py:8]: Hope replacing this to the actual "free of charge" version of the copyright notice is fine.

**Review by Jacob-Scheiffler — APPROVED** [review]

**Comment by sihyung-maxim** [comment]
Don't worry about the failed "verify SVD" action for this PR. This is due to the deprecated register files being updated with the latest copyright notice. The workflow is designed to throw an error if a deprecated file is updated.

File: Examples/MAX32520/SCPA_OTP_Dump/utils.c [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
- * Copyright (C) 2022 Maxim Integrated Products, Inc., All rights Reserved.
+/******************************************************************************
+ * Copyright (C) 2023 Maxim Integrated Products, Inc., All Rights Reserved.
  *
- * This software is protected by copyright laws of the United States and
- * of foreign countries. This material may also be protected by patent laws
- * and technology transfer regulations of the United States and of foreign
- * countries. This software is furnished under a license agreement and/or a
- * nondisclosure agreement and may only be used or reproduced in accordance
- * with the terms of those agreements. Dissemination of this information to
```
Comment (sanjayjaroli) [inline Examples/MAX32520/SCPA_OTP_Dump/utils.c:10]: Utils.c file has special copyright message earlier. Do we really want to update with generic copyright message.

File: Examples/MAX32520/SCPA_OTP_Dump/utils.h [hunk 3]
```diff
@@ -1,15 +1,12 @@
-/*
- ******************************************************************************
- * Copyright (C) 2022 Maxim Integrated Products, Inc., All rights Reserved.
+/******************************************************************************
+ * Copyright (C) 2023 Maxim Integrated Products, Inc., All Rights Reserved.
  *
- * This software is protected by copyright laws of the United States and
```
Comment (sanjayjaroli) [inline Examples/MAX32520/SCPA_OTP_Dump/utils.h:5]: Same as earlier comment.

File: Examples/MAX32650/SCPA_OTP_Dump/utils.h [hunk 4]
```diff
@@ -1,15 +1,12 @@
-/*
- ******************************************************************************
- * Copyright (C) 2022 Maxim Integrated Products, Inc., All rights Reserved.
+/******************************************************************************
+ * Copyright (C) 2023 Maxim Integrated Products, Inc., All Rights Reserved.
  *
- * This software is protected by copyright laws of the United States and
```
Comment (sanjayjaroli) [inline Examples/MAX32650/SCPA_OTP_Dump/utils.h:5]: Same as earlier note.

File: Examples/MAX32670/SecureROM_BL_Host/script/scp_to_c_array.py [hunk 5]
```diff
@@ -1,5 +1,38 @@
...
+ # The mere transfer of this software does not imply any licenses
+ # of trade secrets, proprietary technology, copyrights, patents,
+ # trademarks, maskwork rights, or any other form of intellectual
+ # property whatsoever. Maxim Integrated Products, Inc. retains all
+ # ownership rights.
+ #
+ ###############################################################################
+
+
+copy_right='''/******************************************************************************
```
Comment (sanjayjaroli) [inline Examples/MAX32670/SecureROM_BL_Host/script/scp_to_c_array.py:35]: Duplicate copyright message.

File: Examples/MAX32670/SPI_MasterSlave/main.c [hunk 6]
```diff
@@ -5,38 +5,38 @@
  *          Read the printf() for instructions
```
Comment (sanjayjaroli) [inline Examples/MAX32670/SPI_MasterSlave/main.c:5]: Move example code description after copyright message.

File: Examples/MAX78002/CNN/cifar-100-effnet2/softmax.c [hunk 7]
```diff
@@ -16,9 +16,38 @@
  * limitations under the License.
  */
```
Comment (sanjayjaroli) [inline Examples/MAX78002/CNN/cifar-100-effnet2/softmax.c:17]: Does this file need Maxim Copyright Notice? It has already Arm Limited Copyright Notice.

File: Examples/MAX78002/CNN/kws20_demo/softmax.c [hunk 8]
```diff
@@ -16,9 +16,38 @@
  * limitations under the License.
```
Comment (sanjayjaroli) [inline Examples/MAX78002/CNN/kws20_demo/softmax.c:16]: Same note as earlier for softmax.c

**Pull request merged**